distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc

```diff
@@ -3598,7 +3598,7 @@ ${branding} supports the ability to integrate WFS 1.0 and WFS 2.0 Web Services.
 
 [NOTE]
 ====
-${branding} does not include a supported WFS Web Service (Endpoint) implementation.
+${branding} does not include a WFS Web Service (Endpoint) implementation.
```

Comment:
I know it didn't happen here... but I don't like Endpoint being in ()s.   I always think of things in ()s as things tha could be removed without changing the meaning of the sentence... but without the Endpoint word... this thing is almost saying we support WFS and we don't support it.